Source/Core/DolphinQt2/GameList/GameTracker.cpp

```diff
@@ -79,7 +86,9 @@ void GameTracker::AddDirectoryInternal(const QString& dir)
 void GameTracker::RemoveDirectoryInternal(const QString& dir)
 {
   removePath(dir);
-  QDirIterator it(dir, game_filters, QDir::NoFilter, QDirIterator::Subdirectories);
+  QDirIterator it(dir, game_filters, QDir::NoFilter,
```

Comment:
Would you mind extracting this into a small function that returns the QDirIterator? That way you reduce duplication and I won't have to make an identical change three times when I remove more stuff from SConfig :P

Reply:
Done.